spec/classes/backend_resources_spec.rb

```diff
@@ -8,13 +8,14 @@
         it { should compile }
         it { should have_sensu_namespace_resource_count(1) }
         it { should contain_sensu_namespace('default').with_ensure('present') }
-        it { should have_sensu_user_resource_count(1) }
+        it { should have_sensu_user_resource_count(2) }
```

Comment:
What are the two?

Reply:
One is `agent` and other is `admin` from included `sensu::backend` class.